lib/sengiri/railties/sharding.rake

```diff
@@ -24,6 +24,13 @@ namespace :sengiri do
         rails_env_list.select {|env|
           /_#{env}$/ =~ name
         }.present?
+      }.select {|name|
+        if ENV['SHARD_NAME']
+          env = rails_env_list.select {|env| /_#{env}$/ =~ name }.first
```

Comment:
- `=~` is deprecated in the ruby 2.6.  
  - Please use `match(/../)` ?
  - see [Feature #15231: Remove `Object#=~` - Ruby trunk - Ruby Issue Tracking System](https://bugs.ruby-lang.org/issues/15231)
- 🆓 `.select { .. }.first`  is  `.detect { .. }` .
- 🆓 It looks like indentation is wrong 🤔 
-  🆓I do not think you need to write a lot of .select method

Reply:
Thanks for the comment. Fixed it.